docs/topics/appendix-proficiency-levels.adoc

```diff
@@ -0,0 +1,21 @@
+
+[[proficiency_levels]]
+= Proficiency Levels
+
+Each mission available on {launcher} teaches you about certain topics, but requires certain minimum knowledge, which varies by mission. For clarity, the minimum requirements and concepts are organized in several proficiency levels. In addition to the levels described in this chapter, there can be additional requirements with each mission, specific to its aim or the technologies it uses.
+
+[discrete]
+== {proficiency-foundational}
+
+The missions rated at {proficiency-foundational} proficiency generally require no prior knowledge of the subject matter; they provide general awareness and demonstration of key elements, concepts, and terminology. There are no special requirements except those directly mentioned in the description of the mission.
```

Comment:
I propose to add also "...in the description of the mission or the microservice pattern as (Circuit Breaker, Client and Server Discovery, Application Decomposition, ...)"